builtin/providers/aws/resource_aws_rds_cluster_instance.go

```diff
@@ -83,6 +83,20 @@ func resourceAwsRDSClusterInstance() *schema.Resource {
 				Computed: true,
 			},
 
+			"kms_key_id": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
```

Comment:
But you shouldn't be, right? It's not used in the resource, it's only read. As you noted [in your comment](https://github.com/hashicorp/terraform/issues/7655#issuecomment-232937129) actually using it would error.